### PR TITLE
Book Detail Responsive

### PR DESCRIPTION
**Pull request by joacomiguens**
## Summary

- Book detail responsive.

- Declare $lg variable for breakpoint

## Screenshot

image

## Trello Card

You can find it [here](https://trello.com/c/7go5q5P0/34-responsive)

### CODE REVIEW

File: scss/_detail.scss [hunk 1]
```diff
@@ -3,54 +3,86 @@
...
 .back-button {
   align-self: flex-start;
-  font-size: 20px;
+  font-size: 25px;
   margin: 0 0 40px 40px;
 
+  @media screen and (max-width: $lg) {
+    align-self: center;
+    font-size: 35px;
+    margin: 40px 0 0 0;
```
Comment (ebedoyaalzate) [inline scss/_detail.scss:23]: Just add margin-top

File: scss/_detail.scss [hunk 2]
```diff
@@ -3,54 +3,86 @@
...
 
   &__cover {
-    margin-right: 32px;
-    max-width: 250px;
+    margin-right: 50px;
+    max-width: 300px;
     width: 100%;
+
+    @media screen and (max-width: $lg) {
+      margin: 0 0 40px 0;
```
Comment (ebedoyaalzate) [inline scss/_detail.scss:77]: Just margin-left

File: scss/_variables.scss [hunk 3]
```diff
@@ -1,2 +1,4 @@
-$desktop-width: 1280px;
+$badge-width: 100px;
 $form-border-radius: 5px;
+$lg: 1024px;
```
Comment (ebedoyaalzate) [inline scss/_variables.scss]: Put a more descriptive name

File: scss/_variables.scss [hunk 4]
```diff
@@ -1,2 +1,4 @@
...
+$xlg: 1280px;
```
Comment (ebedoyaalzate) [inline scss/_variables.scss]: Also here

**Review by ebedoyaalzate — CHANGES_REQUESTED** [review]

File: scss/_variables.scss (hunk 3 above)
Comment (joacomiguens) [reply scss/_variables.scss]: Done 😄

File: scss/_variables.scss (hunk 4 above)
Comment (joacomiguens) [reply scss/_variables.scss]: Done 😄

**Review by mcallegari10 — APPROVED** [review]

**Review by ebedoyaalzate — APPROVED** [review]

**Pull request merged**